.github/CONTRIBUTING.md

```diff
@@ -371,13 +371,103 @@ This does not go for bug reports however, as we want to unearth and solve as man
+    - Makes it easier to hand over the issue to other developers if needed
+
+#### The good commit has good commit messages
+
+> Keep in mind: [This](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) &middot; [has](https://www.git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines) &middot; [all](https://github.com/torvalds/subsurface-for-dirk/blob/master/README.md#contributing) &middot; [been](http://who-t.blogspot.co.at/2009/12/on-commit-messages.html) &middot; [said](https://github.com/erlang/otp/wiki/writing-good-commit-messages) &middot; [before](https://github.com/spring-projects/spring-framework/blob/30bce7/CONTRIBUTING.md#format-commit-messages)
+>
+> _&mdash; Chris Beams, [The seven rules of a great Git commit message](https://chris.beams.io/posts/git-commit/#seven-rules)_
+
+All commits from your branch are squashed when merged with the main branch. Commit messages from individual commits end up in a list of changes in the message body of the merge commit. Try to keep that in mind when writing your commit messages.
+
+Ideally, a good commit message will be structured into three parts:
+
+1. Subject
+2. Body
+3. Closing line
+
+The most important part is the subject line. The message body should be used if elaboration is necessary. We rarely use a closing line, but if you want to add useful meta-data related to your commit &ndash; such as GitHub issue number and co-author names &ndash; this is where to put it.
+
+##### A good subject line
+
+* will complete the sentence "If applied, this commit will..."
+* is written in [imperative mood](https://www.thoughtco.com/imperative-mood-grammar-1691151) (_Fix_, not ~~_Fixed_~~, ~~_Fixes_~~ etc.)
+* is limited to 50 characters
+
+If you cannot fit your message into a 50 character subject line, consider if you've included too many changes that makes it difficult to describe concisely. Try to rephrase the subject line and use the [message body](#a-good-message-body) for elaboration.
+
+In addition we use [gitmoji](https://gitmoji.dev) when possible to prefix the subject line with an illustrative emoji. There are several [tools](https://gitmoji.dev/related-tools) available (including a [VSCode extension](https://github.com/vtrois/gitmoji-vscode) and a [CLI tool](https://github.com/carloscuesta/gitmoji-cli)). It's optional, but very helpful when skimming through the git log.
+
+Good words to use as the first of the subject line:
+
+* Add
+* Create
+* Refactor
+* Fix
+* Release
+* Document
+* Modify
+* Update
+* Remove
+* Delete
+
+🟢 Good subject line examples:
+
+> 📝 Update getting started documentation
+
+> 🔥 Remove deprecated methods
+
+> ✅  Add tests for dropdown component
+
+🔴 Bad subject line examples:
+
+> ~~fix a bug.~~
+
+> ~~more changes~~
+
+> ~~added file~~
+
+> ~~fix code review comments~~
+
+##### A good message body
+
+* Separate the message body from the subject line with a blank line.
```

Comment:
The "a good message body" seems a bit tagged on. Which makes sense since (I at least) seldom use message bodies. If you agree with me, perhaps introduce "a good message body" with something like: 

> While we seldom use message bodies, we encourage you to do so if you find it necessary. Here is some pointers, on how to write a good one:

Reply:
PS. All in for throwing in that migmig!